src/components/wizards/new-vm/NewVmWizard.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Wizard, Button } from 'patternfly-react';
+import { CreateVmWizard } from '../create-vm/CreateVmWizard';
+import { ImportVmWizard } from '../import-vm/ImportVmWizard';
+import './NewVmWizard.css';
+
+export class NewVmWizard extends React.Component {
+
+    state = {
+      createVM: false,
+      importVM: false
+    }
+  
+    openCreateVmWizard = () => this.setState({createVM: true});
+
+    openImportVmWizard = () => this.setState({importVM: true});
+  
+    render() {
+      let wizard;
+      if(this.state.createVM) {
+        wizard = <CreateVmWizard onHide={this.props.closeWizard} {...this.props}/>
+      } else if(this.state.importVM) {
+        wizard = <ImportVmWizard onHide={this.props.closeWizard} {...this.props}/>
+      }
+      return (
+        <React.Fragment>
+          <Wizard show onHide={this.props.closeWizard}>
+            <Wizard.Header title="Create Virtual Machine" onClose={this.props.closeWizard} />
+            <Wizard.Body>
+              <Wizard.Row>
+                <Wizard.Main>
+                  <Wizard.Contents stepIndex={0} activeStepIndex={0} className="wizard-content">
+                    <button className="btn btn-link wizard-button " onClick={this.openCreateVmWizard}>
+                      <div className="col-md-12">
+                        <span className="pficon-virtual-machine fa-5x" />
+                      </div>
+                      <div className="col-md-12">
+                        <span className="lead">Create New<br />Virtual Machine</span>
+                      </div>
+                    </button>
+                    <button className="btn btn-link wizard-button" onClick={this.openImportVmWizard}>
+                      <div className="col-md-12">
+                        <span className="pficon-import fa-5x" />
+                      </div>
+                      <div className="col-md-12">
+                        <span className="lead">Import Existing<br />Virtual Machine</span>
+                      </div>
+                    </button>
+                  </Wizard.Contents>
+                </Wizard.Main>
+              </Wizard.Row>
+            </Wizard.Body>
+            <Wizard.Footer>
+              <Button bsStyle="default" className="btn-cancel" onClick={this.props.closeWizard}>Cancel</Button>
+            </Wizard.Footer>
+          </Wizard>
+          {wizard}
+        </React.Fragment>
+      );
+    }
+}
```

Comment:
Are we using React Proptypes in this project or not ?

Reply:
we should, :+1:

Reply:
Of course we should use them :smiley: maybe not right now, since we're prototyping our first complex wizard component, but we should use them at some point.